modules/planning/common/frame_open_space.h

```diff
@@ -89,6 +92,16 @@ class FrameOpenSpace {
 
   const bool is_near_destination() const { return is_near_destination_; }
 
+  const std::size_t nOb() { return nOb_; }
+
+  const Eigen::MatrixXd &vOb() { return vOb_; }
```

Comment:
Please use name that is optimized for the reader rather than for the writer. Use full names when possible unless commonly accepted abbreviations.